gslib/tests/test_defacl.py

```diff
@@ -215,6 +226,70 @@ def testTooFewArgumentsFails(self):
     # Neither arguments nor subcommand.
     stderr = self.RunGsUtil(['defacl'], return_stderr=True, expected_status=1)
     self.assertIn('command requires at least', stderr)
+  
+class TestDefaclShim(case.GsUtilUnitTestCase):
+  
+
+  @mock.patch.object(defacl.DefAclCommand, 'RunCommand', new=mock.Mock())
+  def test_shim_translates_defacl_get(self):
+    with SetBotoConfigForTest([('GSUtil', 'use_gcloud_storage', 'True'),
+                               ('GSUtil', 'hidden_shim_mode', 'dry_run')]):
+      with SetEnvironmentForTest({
+          'CLOUDSDK_CORE_PASS_CREDENTIALS_TO_GSUTIL': 'True',
+          'CLOUDSDK_ROOT_DIR': 'fake_dir',
+      }):
+        mock_log_handler = self.RunCommand('defacl', ['get', 'gs://bucket'],
+                                           return_log_handler=True)
+        info_lines = '\n'.join(mock_log_handler.messages['info'])
+        self.assertIn(
+            ('Gcloud Storage Command: {} storage buckets describe'
+             ' --format=multi(defaultObjectAcl:format=json) gs://bucket').format(
+                 os.path.join('fake_dir', 'bin', 'gcloud')), info_lines)
+
+  @mock.patch.object(defacl.DefAclCommand, 'RunCommand', new=mock.Mock())
+  def test_shim_translates_set_defacl_file(self):
+    acl_string = '''[{
```

Comment:
The temp file doesn't actually need any contents b/c it just has to exist for `os.path.isfile` to pass

Reply:
Deleted temp file content